SYCL/SubGroupMask/Simple.cpp

```diff
@@ -0,0 +1,255 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// REQUIRES: gpu
+// GroupNonUniformBallot capability is supported on Intel GPU only
```

Comment:
does it mean that we should not run this test on HIP and CUDA?

Reply:
Good point, that comes from the other tests but is incorrect now, this test should work fine on CUDA and HIP, I updated the test to remove this comment.